### DISCUSSION TITLE
Writing OpenROAD logs into a file

### DISCUSSION THREAD

**Discussion by lustefan97** [The-OpenROAD-Project/OpenROAD#4134] [Q&A] [answered]
Hello,

I would like to know if there is any way to write the log output from OpenROAD into a file (using a report_ type command). I am using a custom script, not the ORFS flow, so I would like to be able to store every tool report into a file to parse it using a script and find the information that I need.

I know that the ORFS flow does something similar, but as I am using a custom script I would like to do know if OpenROAD allow to do so or if I need to find another way.

**Comment by maliberty** [comment, marked as answer]
The output is a stdout so you can use standard unix file operations to capture it.  See https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/6c4437edabdaddde9f100d76cc77b568c6cc073d/flow/Makefile#L539 in ORFS where we use "2>&1 | tee"
